includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```diff
@@ -1668,9 +1881,11 @@ private function get_customer_id_for_order( WC_Order $order ): string {
 	 * Returns the statement descriptor given the selected payment type.
 	 *
 	 * @param string $selected_payment_type The selected payment type.
+	 * @param WC_Order $order The WC order for which we're getting the statement descriptor.
```

Comment:
```suggestion
	 * @param WC_Order $order The WC order for which we're getting the statement descriptor.
	 * @param string $selected_payment_type The selected payment type.
```

The order of these params don't match the function signature. `$order` is the first param.

Reply:
Good catch. It's updated now, thanks!